src/gg.regression.unity.experimental.runespawn/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "gg.regression.unity.experimental.runespawn",
+    "version": "0.0.28",
+    "description": "An experimental package that focuses on code generation.",
```

Comment:
I would add this text to the end of the description, similar/same to our ecs extension

  To use this package, please also add the `gg.regression.unity.bots` package from URL `https://github.com/Regression-Games/RGUnityBots.git?path=src/gg.regression.unity.bots#v0.0.28`.